inc/options/class-wpseo-option-wpseo.php

```diff
@@ -119,6 +124,58 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Add filters to make sure that the option is merged with its defaults before being returned.
+	 *
+	 * @return void
+	 */
+	public function add_option_filters() {
+		parent::add_option_filters();
+
+		// Add filter to ensure the feature variables respect the network settings.
+		if ( has_filter( 'option_' . $this->option_name, array( $this, 'verify_features_against_network' ) ) === false ) {
```

Comment:
You might consider to put the value `options_ . $this->options_name` in a variable.